Gulpfile.js

```diff
@@ -199,9 +214,10 @@ function getMatchingFiles(aFileArgs, aFileType)
     return lMatchingFiles;
 }
 
-function getAllTestFiles(aDirectory)
+function getAllTestFiles(aDirectory, aFilter)
 {
     const lFiles = [];
+    aFilter = aFilter || "test.js" 
```

Comment:
Two things:
 1. Use `??` instead of `||` in line with our `prefer-nullish` lint rule
 2. Does the suggested change also work / preferable?

```suggestion
function getAllTestFiles(aDirectory, aFilter = "test.js")
{
    const lFiles = [];
```